docs/select/README.md

```diff
@@ -1,20 +1,17 @@
 # Select API Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 Traditional retrieval of objects is always as whole entities, i.e GetObject for a 5 GiB object, will always return 5 GiB of data. S3 Select API allows us to retrieve a subset of data by using simple SQL expressions. By using Select API to retrieve only the data needed by the application, drastic performance improvements can be achieved.
 
-> This implementation is compatible with AWS S3 Select API
+You can use Select API to query objects that have the following format properties:
 
-### Implemention status:
-
-- Full S3 SQL syntax is supported
-- All aggregation, conditional, type-conversion and strings SQL functions are supported
-- JSONPath expressions are not yet evaluated
-- Large numbers (more than 64-bit) are not yet supported
-- Date related functions are not yet supported (EXTRACT, DATE_DIFF, etc)
-- S3's reserved keywords list is not yet respected
+- CSV, JSON and Parquet - Objects must be in CSV, JSON, or Parquet format.
+- UTF-8 is the only encoding type Select API supports.
+- GZIP or BZIP2 - CSV and JSON files can be compressed using GZIP or BZIP2. Select API supports columnar compression for Parquet using GZIP, Snappy, LZO, LZ4. Whole object compression is not supported for Parquet objects.
+- Server-side encryption - Select API supports querying objects that are protected with server-side encryption.
+- CAST function if defined is honored, if CAST is not provided then type is inferred for such as decimal values etc.
```

Comment:
This line does not belong in a list of object properties. We need to set it out of the list, and rephrase it like:

> Type inference and automatic conversion of values is performed based on the context when the value is un-typed (such as when reading CSV data). The behavior can be disabled by providing explicit casts using the CAST function.

Reply:
Can you provide a suggestion @donatello instead?

Reply:
Anyways I updated.